### DISCUSSION TITLE
How do I connect the WIFI client to a network that requires a username and password

### DISCUSSION THREAD

**Discussion by arohl** [RaspAP/raspap-webgui#1557] [Q&A]
I am trying to connect to a wireless network that requires a username and a password. The network is discovered in the WIFI client but there is no field for the username. Is it possible to connect to this network?

Many thanks

  Andrew

**Comment by billz** [comment]
This is a university network, correct? Higher ed institutions on the eduroam network (a popular one) use WPA2-Enterprise with 802.1x/PEAP authentication. This is not currently supported by RaspAP. However, you should be able to connect with a manual wpa_supplicant configuration. Campus IT help desk can advise you here.
